encode-faq-titles.sh

```diff
@@ -1,15 +1,18 @@
 #!/bin/bash
 set -e
 
-if [ -z "${AUTH_PASSWORD}" ]; then
+if [ -z "${AUTH_PASSWORD}" ] && [[ "$@" != *"-local"* ]]; then
     echo "no AUTH_PASSWORD provided!"
     exit 1
 fi
```

Comment:
Maybe this if should just be moved to nested into the else below then won't need to check local again?